hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/routes.json

```diff
@@ -23,5 +23,9 @@
   "/namespace/summary?path=*": "/metadata",
   "/namespace/quota?path=*": "/quota",
   "/task/status": "/taskStatus",
-  "/containers/unhealthy": "/unhealthyContainers"
+  "/containers/unhealthy": "/unhealthyContainers",
+  "/heatmap/readaccess?startDate=90D&path=*&entityType=*": "/heatmapTestClick",
```

Comment:
what this is used for ? "heatmapTestClick"

Reply:
is this necessary? I can't see it used anywhere else